src/MySqlConnector/MySqlConnector.csproj

```diff
@@ -10,6 +10,8 @@
     <PackageId>MySqlConnector</PackageId>
     <PackageTags>mysql;mysqlconnector;async;ado.net;database;netcore</PackageTags>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <AssemblyVersion>0.45.3.0</AssemblyVersion>
+    <Version>0.45.3</Version>
```

Comment:
Please delete these lines; I will update the version as necessary before shipping the next version.